include/deal.II/base/table.h

```diff
@@ -2367,6 +2624,168 @@ Table<2,T>::n_cols () const
 
 
 
+//---------------------------------------------------------------------------
+namespace TransposeTableIterators
+{
+  template <typename T, bool Constness>
+  inline
+  AccessorBase<T, Constness>::AccessorBase()
+    :
+    container(nullptr),
+    linear_index(std::numeric_limits<decltype(linear_index)>::max())
+  {}
+
+
+
+  template <typename T, bool Constness>
+  inline
+  AccessorBase<T, Constness>::AccessorBase(const container_pointer_type table)
+    :
+    container(table),
+    linear_index(container->values.size())
```

Comment:
Do you think of this as the end iterator then?

Reply:
Yes (see the constructor documentation :))